docs/maintain/hornet/2.0/docs/how_tos/using_docker.md

```diff
@@ -51,6 +51,14 @@ It includes everything required to setup a public node accessible by wallets and
 
 5. [curl](https://curl.se/).
 
+## Recommended Hardware Requirements
+
+We recommend that you run the docker image on a server with:
+
+* **CPU**: 8 core.
+* **RAM**: 16 GB.
+* **Disk space**: 250 GB SSD.
+* 
```

Comment:
These are not Hornet requirements. Hornet requirements are mentioned [here](https://wiki.iota.org/hornet/getting_started/#recommended-requirements)

These are Wasp requirements and I would add them [here](https://wiki.iota.org/wasp/how-tos/running-a-node/)

Also the disk space should not just be a number. We should mention that it depends on your pruning config